cmd/juju/storage/volumelistformatters.go

```diff
@@ -70,9 +69,9 @@ func formatVolumeListTabular(writer io.Writer, infos map[string]VolumeInfo) erro
 	sort.Sort(volumeAttachmentInfos)
 
 	if haveMachines {
-		print("Machine", "Unit", "Storage", "Id", "Provider Id", "Device", "Size", "State", "Message")
+		print("Machine", "Unit", "Storage Id", "Volumes Id", "Provider Id", "Device", "Size", "State", "Message")
```

Comment:
Where we have "Storage Id", it needs to be "Storage id". Same for Volume Id etc. Also not Volumes
Looks like the original code here was wrong.